packages/plugins/documentation/server/middlewares/documentation.js

```diff
@@ -22,4 +22,15 @@ module.exports = async ({ strapi }) => {
       },
     },
   ]);
+
+  const customPath = strapi.plugin('documentation').config('x-strapi-config.path');
+
+  if (customPath) {
```

Comment:
This is always true because the default configuration for the plugin has a `path` in it.

I could skip this step when `customPath` is equal to `/documentation`, but I figured it'd be better to not hardcode that `/documentation` there.

Let me know what you guys think.